src/test/java/org/opentripplanner/CoordinateHelper.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner;
+
+import org.opentripplanner.framework.geometry.SphericalDistanceLibrary;
+import org.opentripplanner.framework.geometry.WgsCoordinate;
+
+/**
+ * Helper class for creating coordinates at reasonable distances from each other.
+ */
+public class CoordinateHelper {
```

Comment:
This should probably go here: https://github.com/opentripplanner/OpenTripPlanner/tree/711c59d819afdb8a741bcf7ac3ed2654d7f246d9/src/test/java/org/opentripplanner/_support/geometry